pipeline/test/services/test-sc.sh

```diff
@@ -14,10 +14,10 @@ DEBUG_OUTPUT=("")
 DEBUG_PROMETHEUS_TARGETS=("")
 export CLUSTER="ServiceCluster"
 
-source "${SCRIPTS_PATH}"/service-cluster/testPodsReady.sh
-source "${SCRIPTS_PATH}"/common/testPersistentVolumeClaims.sh
-source "${SCRIPTS_PATH}"/service-cluster/testEndpoints.sh
-source "${SCRIPTS_PATH}"/service-cluster/testPrometheusTargets.sh
+"${SCRIPTS_PATH}/service-cluster/testPodsReady.sh"
+"${SCRIPTS_PATH}/common/testPersistentVolumeClaims.sh"
+"${SCRIPTS_PATH}/service-cluster/testEndpoints.sh"
+"${SCRIPTS_PATH}/service-cluster/testPrometheusTargets.sh"
```

Comment:
Don't change this, otherwise the count does not work.

Reply:
Ah, my bad. I kind of dislike using source for executing things but I see why it's being used here. What do you think of sourcing the files and executing the functions inside the `test-*.sh` scripts instead?

Reply:
Yeah I understand that. I don't really like this either.
Not sure I fully like your suggestion either, but I'm ok with it and I don't think I have any better suggestions. If you go with that, then I suggest that you execute each function directly below the line that sources them. So that it's clear which file a function is coming from.

Reply:
I saw the execution inside the testUserRbac.sh script and it's way too much to clean up right now to be bothered. I think all of the tests under `test/services` really should live in `bin/`  because all of that really is the implementation of the `ck8s test` command and not for the pipeline only.

Reply:
Yeah I agree that we probably should not have our tests inside `pipeline/`. Maybe we can move them to another folder (`tests/`?), but that is for another PR. You can create an issue if you want.